src/scenes/home/informationForm/formComponents/identifier.js

```diff
@@ -16,18 +22,28 @@ class Identifier extends Component {
           prompt="Which describes you"
           onChange={e => this.props.update(e, e.target.value)}
           className={styles.signup}
+          validationFunc={e => this.props.validationFunc(e)}
         />
+        { !this.props.isValid && this.props.validationErrorMessage }
```

Comment:
This may seem redundant, but what's the purpose of this conditional? What's it doing for us here?

Reply:
simply checking for the validity of the component value.
in case it's in valid we want to show the error message, on the other hand, if it's valid we don't show anything due to the `and` operator.

Reply:
Awesome. Thanks for the clarification there. =D

Reply:
no problem!